docs/root/intro/arch_overview/websocket.rst

```diff
@@ -20,6 +20,31 @@ one can set up custom
 for the given upgrade type, up to and including only using the router filter to send the WebSocket
 data upstream.
 
+Handling H2 hops (implementation in progress)
+---------------------------------------------
+
+One oft requested feature for Envoy was to allow WebSocket to traverse HTTP/2 hops, where there
```

Comment:
:). I would remove the origin of this feature and simply state it as a fact like "Envoy currently has an alpha implementation of tunneling websockets over H2 streams for deployments that prefer a uniform H2 mesh throughout."